plugins/mac-capture/mac-sck-common.h

```diff
@@ -28,6 +28,8 @@ typedef enum {
     ScreenCaptureAudioApplicationStream = 1,
 } ScreenCaptureAudioStreamType;
 
+typedef SCDisplay *SCDisplayRef;
+
```

Comment:
I'm surprised to see clang-format adding whole new lines? Or is this a manual fix?

Reply:
See above.